### PR TITLE
Snap banner UI

### PR DESCRIPTION
**Pull request by abhishek-01k**
https://www.figma.com/file/PzAmXUMOmwS2ljxufwv44m/PUSH-Prototypes?type=design&node-id=19055-29518&mode=design&t=AOL47VeLIBFytYcG-0

Link to figma design

### CODE REVIEW

File: src/modules/snap/SnapModule.tsx [hunk 1]
```diff
@@ -19,11 +20,13 @@ import SnapExample from 'assets/PushSnaps/SnapExample.svg';
 import InfoLogo from 'assets/PushSnaps/spam-icon.svg';
 import PushMetamaskLogo from 'assets/PushSnaps/PushMetamaskLogo.svg';
 import ActiveIcon from 'assets/PushSnaps/ActiveIcon.svg';
+import NotificationLogo from 'assets/PushSnaps/Notification.svg';
+import BellRinging from 'assets/PushSnaps/BellRinging.svg';
+import GasPump from 'assets/PushSnaps/GasPump.svg';
+import AboutSnapModal from './AboutSnapModal';
 import { ReactComponent as Gear } from 'assets/PushSnaps/Gear.svg';
+import { useNavigate } from 'react-router-dom';
```
Comment (arn4b) [inline src/modules/snap/SnapModule.tsx:28]: move this import to the top with other package imports

**Review by arn4b — APPROVED** [review]
LGTM

**Comment by abhishek-01k** [comment]
Screenshot 2024-01-03 at 5 42 13 PM

**Review by route-2 — APPROVED** [review]

**Review by arn4b — APPROVED** [review]
LGTM

**Review by 0xNilesh — APPROVED** [review]

**Pull request merged**